### PR TITLE
Regenerated reference for kubeadm

### PR DESCRIPTION
**Pull request by tengqm**
The upstream source code now applies normalization function to all
command line flags. The PR captures such a change for all references
impacted.

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/approve

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-27818--kubernetes-io-master-staging.netlify.app/docs/reference/setup-tools/kubeadm/kubeadm-certs/#options-inherited-from-parent-commands

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm.md [hunk 1]
```diff
@@ -56,20 +56,118 @@ Example usage:
 </colgroup>
 <tbody>
 
+<tr>
+<td colspan="2">--add-dir-header</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>If true, adds the file directory to the header of the log messages</p></td>
```
Comment (kbhawkey) [inline content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm.md:63]: Hi @tengqm .
Did the upstream code change (post v1.21.0) and now you can generate the "Optional parent flags" for each command?
It seems somewhat repetitive. Do all commands share these common or parent commands (or most commands)?

**Comment by kbhawkey** [comment]
Thanks @tengqm 
/lgtm
Hold for review
/hold
/cc @neolit123

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm.md:63]: Yes.

It looks like most sub-commands share the same set of parent command options.

**Comment by kbhawkey** [comment]
Still sorting out flags
/lgtm cancel

**Comment by neolit123** [comment]
Lgtm.
I see the inherited klog flags are now gone too, but maybe that was the
intent.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-27818--kubernetes-io-master-staging.netlify.app/docs/reference/setup-tools/kubeadm/kubeadm-init-phase/#options

**Pull request merged**